src/toxcore/dht/server/mod.rs

```diff
@@ -220,9 +220,16 @@ impl Server {
     }
 
     /// Add a friend.
-    pub fn add_friend(&self, friend: DhtFriend) {
+    /// `node_to_bootstrap` of new friend is filled with close nodes for fast bootstrapping.
+    pub fn add_friend(&self, mut friend: DhtFriend) {
         let mut friends = self.friends.write();
 
+        let close_nodes = self.close_nodes.read().get_closest(&friend.pk, true);
```

Comment:
I suggest to unify with: https://github.com/tox-rs/tox/blob/64d49ce2730ff30826ef9f33b66824f24c3bda88/src/toxcore/dht/server/mod.rs#L770-L792

i.e. create method in the server that gets closest nodes from both `close_nodes` and `friends` and use it here as well.